src/Mod/Draft/InitGui.py

```diff
@@ -92,25 +92,47 @@ def QT_TRANSLATE_NOOP(context, text):
 
         # Set up command lists
         import draftutils.init_tools as it
-        self.drawing_commands = it.get_draft_drawing_commands()
+        if hasattr(it, "get_draft_drawing_commands_temp"):
```

Comment:
Should there be a note here about removing when BIM is updated?

Reply:
Actually I was mistaken that this mod affects the BIM workbench. Only the Arch workbench relies on this code. I'll improve the PR with that in mind.